awx/main/management/commands/setup_managed_credential_types.py

```diff
@@ -10,4 +10,4 @@ class Command(BaseCommand):
     help = 'Load default managed credential types.'
 
     def handle(self, *args, **options):
-        CredentialType.setup_tower_managed_defaults()
+        CredentialType.setup_tower_managed_defaults(wait_for_lock=True)
```

Comment:
If someone is explicitly running this management command then they probably want to make sure the code runs to register the CredentialType. Maybe they are debugging an error and want to trigger the code path.

Reply:
is the ready() being run before or after this?

Reply:
might want to leave some comments around this command its kinda... vestigial

Reply:
Comment added.